src/utils/generateChartData.js

```diff
@@ -126,21 +126,22 @@ const generateChartData = (data, metrics, metricLabels = []) => {
   );
   const { startYear, startMonth, endYear, endMonth } = periods;
 
-  let i = startYear * 12 + startMonth;
-  const lastMonth = endYear * 12 + endMonth;
+  let i = annual ? startYear - 5 : startYear * 12 + startMonth;
```

Comment:
This bit I don't totally understand. Why do we need to subtract 5 here? Is it to ensure that there are always at least 5 labels to display in the 5 year chart, and if so why don't we need to do something similar for the monthly data?

Reply:
For monthly data we calculate the amount of months, and for annual we need this to display dates with empty data, because without this,  graph will show just previous months of last reported year.